net/lighttpd-https-cert/files/ssl-enable.conf

```diff
@@ -4,14 +4,22 @@
 # This package is not meant to be edited, it is part of package lighttpd-https-cert
 # If you need different https setup, uninstall this package and create your own configuration
 
+# As a special case in lighttpd 1.4.46 and later, if ssl.engine = "enable" in
+# the global scope and along with ssl.pemfile, then $SERVER["socket"] blocks
+# can inherit the exact global config by setting ssl.engine = "enable" and no
+# other ssl.* directives in the $SERVER["socket"] block.
```

Comment:
I like this feature and explanation but I think the comment should not be here but in the Pull Request description and/or in the commit message.

Reply:
See https://github.com/CZ-NIC/turris-os-packages/pull/59/commits/ab095c73aca4eae30096cd9ca63ef4b1629f688b